libs/platform/src/lib/components/approval-flow/approval-flow.module.ts

```diff
@@ -2,25 +2,53 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
```

Comment:
Have this weird drop area collisions:

![image](https://user-images.githubusercontent.com/3263744/107957602-8109ee80-6fa9-11eb-86b8-a0aad677bd5a.png)

Reply:
Is it by design?

Reply:
this might seem weird but technically works ok - a drop zone is highlighted if we drag a node over it, and even if multiple drop zones are active, only the first one (top-right) is used. This needs further investigation and discussion with UI/UX team so I think current behavior is ok for the Phase 2